test/modules/core/lib/attribute/attribute.spec.js

```diff
@@ -422,6 +422,81 @@ test('Attribute#updateBuffer', t => {
   t.end();
 });
 
+test('Attribute#standard accessor - variable width', t => {
+  const TEST_PROPS = {
+    data: [
+      {id: 'A', value: [10, 11], color: [[255, 0, 0], [255, 255, 0]]},
+      {id: 'B', value: [20], color: [[128, 128, 128], [128, 128, 128]]},
+      {id: 'C', value: [30, 31, 32], color: [255, 255, 255]}
+    ],
+    getColor: d => d.color,
+    getValue: d => d.value
+  };
+
+  const TEST_CASES = [
+    {
+      attribute: new Attribute(gl, {
+        id: 'values',
+        type: GL.FLOAT,
+        size: 1,
+        accessor: 'getValue'
+      }),
+      result: [10, 11, 20, 30, 31, 32]
+    },
+    {
+      attribute: new Attribute(gl, {
+        id: 'colors',
+        type: GL.UNSIGNED_BYTE,
+        size: 4,
+        defaultValue: [0, 0, 0, 255],
+        accessor: 'getColor'
+      }),
+      result: [
+        255,
+        0,
+        0,
+        255,
+        255,
+        255,
+        0,
+        255,
+        128,
+        128,
+        128,
+        255,
+        255,
+        255,
+        255,
```

Comment:
`[255, 255, 255, 255]` is default value of a color attribute?

Reply:
defaultValue is assigned in line 451.